docs/serving/istio-authorization.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Enabling requests to Knative services when additional authorization policies are enabled"
```

Comment:
Is this a doc for specifically the case when additional authorization policies are enabled or is it a general doc for using Istio authorization  with Knative and Istio mesh mode (per the PR title)?

Reply:
Yes, It is a general doc for using Istio authorization with Knative and Istio mesh mode.